static/js/message_list.js

```diff
@@ -507,16 +507,16 @@ exports.MessageList.prototype = {
     },
 
     hide_edit_message: function MessageList_hide_edit_message(row) {
-        row.find(".message_content, .status-message").show();
-        row.find(".message_edit").hide();
+        row.find(".message_content, .status-message").show(150, "linear");
```

Comment:
I think we can go here a bit slower as well, I tried out values between 150-250 and 225 seemed a nice one though we can experiment a bit here as well. I feel we should at least have a 200 here.
This should apply to `show_edit_topic` as well.